Kconfig

```diff
@@ -1109,3 +1110,5 @@ menu "LVGL configuration"
     endmenu
 
 endmenu
+endif
+
```

Comment:
...ok now you have the newline, but you also have an empty line :-) can you drop that? Guess the editor should do the right thing at this point